### PR TITLE
When creating a workshop, 'funded' should be a dropdown and should be…

### PR DESCRIPTION
**Pull request by clareconstantine**
… required

[Trello card](https://trello.com/c/oBX2iQYm/172-when-creating-a-workshop-creator-must-select-funded-or-unfunded-default-to-unselected)

**Before:**

screen shot 2017-10-02 at 3 48 20 pm

**After:**
Default is unselected, and the field is required.

screen shot 2017-10-02 at 3 49 39 pm

screen shot 2017-10-02 at 4 02 34 pm

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx [hunk 1]
```diff
@@ -600,6 +592,11 @@ const WorkshopForm = React.createClass({
         validation.style.subject = "error";
         validation.help.subject = "Required.";
       }
+      if (!this.state.funded) {
+        validation.isValid = false;
+        validation.style.funded = "error";
+        validation.help.funded = "Required";
+      }
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx]: Won't this fail validation whenever it's false? Shouldn't this be `this.state.funded === null` ?

Comment (clareconstantine) [reply apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx]: yes! i forgot to change this when we switched it away from strings. thanks!

**Comment by clareconstantine** [comment]
@tanyaparker here is the new dropdown!

**Comment by tanyaparker** [comment]
Woo! Sorry to be unclear in my mockups, but can we change the strings to be super clear:

Yes, it is funded.
No, it is not funded.

**Comment by tanyaparker** [comment]
[When creating a workshop, creator must select Funded or Unfunded (default to unselected)](https://trello.com/c/oBX2iQYm/172-when-creating-a-workshop-creator-must-select-funded-or-unfunded-default-to-unselected)

**Pull request merged**